src/backend/x11.rs

```diff
@@ -102,26 +109,30 @@ pub fn create_screenshots() -> Result<Vec<(OutputInfo, image::DynamicImage)>, Er
     Ok(images)
 }
 
-fn get_image(conn: &RustConnection, screen: &Screen) -> Result<DynamicImage, Error> {
+fn get_image(
+    conn: &RustConnection,
+    screen: &Screen,
+    x: i16,
+    y: i16,
+    width: u16,
+    height: u16,
+) -> Result<DynamicImage, Error> {
     use x11rb::protocol::xproto::ConnectionExt;
     const ALL_BITS: u32 = u32::MAX;
 
     let setup = &conn.setup();
-    let width_u16 = screen.width_in_pixels;
-    let height_u16 = screen.height_in_pixels;
-
-    let width_u32 = u32::from(width_u16);
-    let height_u32 = u32::from(height_u16);
+    let width_u32 = u32::from(width);
+    let height_u32 = u32::from(height);
 
     let (image_bytes, pixmap_format) = {
         let image_reply = conn
             .get_image(
                 ImageFormat::Z_PIXMAP,
                 screen.root,
-                0,
-                0,
-                width_u16,
-                height_u16,
+                x,
+                y,
+                width,
+                height,
```

Comment:
oh damn... good catch! Sorry about that...

Reply:
It's fine, we do all make mistakes.